WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletView.xaml

```diff
@@ -31,6 +31,7 @@
         </Button>
         <Button Content="Search Hardware Wallets" Command="{Binding EnumerateHardwareWalletsCommand}" DockPanel.Dock="Right" Margin="0 0 10 0" />
         <Button Content="Import Coldcard" Command="{Binding ImportColdcardCommand}" DockPanel.Dock="Right" Margin="0 0 10 0" />
+        <Button Content="Import Cobo Vault" Command="{Binding ImportCoboVaultCommand}" DockPanel.Dock="Right" Margin="0 0 10 0" />
```

Comment:
Thanks for the PR!

I cannot test it, don't yet have the hww...

Since both ColdCard & CoboVault have the same UX, I think we can just have one button with `Import hardware wallet` or `Import skelethon file` or something. No need to have a unique button for each wallet provider when the UX is the same..

Reply:
Sounds good 👍